modules/core/src/lib/composite-layer.js

```diff
@@ -18,9 +18,12 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 import Layer from './layer';
-import log from '../utils/log';
+import debug from '../debug';
+const debugLog = debug.log;
```

Comment:
Mixing statements and imports typically does not agree with modern linter settings... If possible avoid it.